specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -7557,6 +7569,319 @@
           }
         }
       }
+    },
+    "/apps/{appId}/azureaccounts": {
+      "post": {
+        "description": "Assigns an azure account to the application.",
+        "operationId": "5be32228e8473de116325515",
```

Comment:
Change operationId to something readable like MainSection_ApiName as Apps_Add with MainSection and ApiName format in upper camel-case